### PR TITLE
Edge troubleshooting

### PR DESCRIPTION
**Pull request by Rick-Kirkham**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[2e2488e](https://github.com/OfficeDev/office-js-docs-pr/commits/2e2488e5b2ff9fdee7506982b63f5abc63b3987b)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/browsers-used-by-office-web-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/edge-troubleshooting/docs/concepts/browsers-used-by-office-web-add-ins.md) | :warning:Warning | | [Details](#user-content-ffbf8e59-33f99382)

<a id="ffbf8e59-33f99382"></a>
### [docs/concepts/browsers-used-by-office-web-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/edge-troubleshooting/docs/concepts/browsers-used-by-office-web-add-ins.md)
 - **Line 60**: **[Warning]** `````Invalid file link:(~/../troubleshoot/error-messages/cannot-open-add-in-from-localhost).`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C9%5C26%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201909260011126748-1328%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=etIxg%2F2OcFtV%2BUNMQUsSqRBTvSJUv88VcpPmh%2BI2yo8%3D&st=2019-09-26T00%3A07%3A25Z&se=2019-10-27T00%3A12%3A25Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[309759c](https://github.com/OfficeDev/office-js-docs-pr/commits/309759ced301779425d02c632633aedaa1e15c0e)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/browsers-used-by-office-web-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/edge-troubleshooting/docs/concepts/browsers-used-by-office-web-add-ins.md) | :warning:Warning | | [Details](#user-content-ffbf8e59-e71e2f67)

<a id="ffbf8e59-e71e2f67"></a>
### [docs/concepts/browsers-used-by-office-web-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/edge-troubleshooting/docs/concepts/browsers-used-by-office-web-add-ins.md)
 - **Line 60**: **[Warning]** `````Invalid file link:(~/../../troubleshoot/error-messages/cannot-open-add-in-from-localhost).`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C9%5C26%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201909260409108480-1328%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=euT0mXmiGdudENSeweTE%2BBB7u2soNeEk5TfZPLKLUvs%3D&st=2019-09-26T04%3A05%3A29Z&se=2019-10-27T04%3A10%3A29Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/concepts/browsers-used-by-office-web-add-ins.md [hunk 1]
```diff
@@ -45,6 +45,21 @@ The following table shows which browser is used for the various platforms and op
 > 1. Go to [Get started as an Office Insider](https://insider.office.com/join).
 > 2. Follow the instruction on that page to join. When asked to specify a channel, select Insider.
 
+## Troubleshooting Microsoft Edge Issues
+
+### Scroll bar does not appear in task pane
+
+By default, scroll bars in Microsoft Edge are hidden until hovered over. To ensure that the scroll bar is always visible, the CSS styling that applies to the `<body>` element of the pages in the task pane should include the [-ms-overflow-style](https://developer.mozilla.org/en-US/docs/Web/CSS/-ms-overflow-style) property and it should be set to `scrollbar`.
```
Comment (ElizabethSamuel-MSFT) [inline docs/concepts/browsers-used-by-office-web-add-ins.md]: Recommend removing locale from URL
```suggestion
By default, scroll bars in Microsoft Edge are hidden until hovered over. To ensure that the scroll bar is always visible, the CSS styling that applies to the `<body>` element of the pages in the task pane should include the [-ms-overflow-style](https://developer.mozilla.org/docs/Web/CSS/-ms-overflow-style) property and it should be set to `scrollbar`. 
```

File: docs/concepts/browsers-used-by-office-web-add-ins.md [hunk 2]
```diff
@@ -45,6 +45,21 @@ The following table shows which browser is used for the various platforms and op
...
+
+### When debugging with the Microsoft Edge DevTools, the add-in crashes or reloads
+
+Setting breakpoints in the [Microsoft Edge DevTools](https://www.microsoft.com/p/microsoft-edge-devtools-preview/9mzbfrmz0mnj?rtc=1&activetab=pivot%3Aoverviewtab) can cause Office to think that the add-in is hung. It will automatically reload the add-in when this happens. To prevent this, add the following Registry key and value to the development computer: `[HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Office\16.0\Wef]"AlertInterval"=dword:00000000`.
+
+### When the add-in tries to open, get "ADD-IN ERROR We can't open this add-in from the localhost" error
+
+One known cause is that Microsoft Edge requires that localhost be given a loopback exemption on the development computer. Follow the instructions at [Cannot open add-in from localhost](../../../troubleshoot/error-messages/cannot-open-add-in-from-localhost).
```
Comment (ElizabethSamuel-MSFT) [inline docs/concepts/browsers-used-by-office-web-add-ins.md]: This should fix the link issue...

```suggestion
One known cause is that Microsoft Edge requires that localhost be given a loopback exemption on the development computer. Follow the instructions at [Cannot open add-in from localhost](/office/troubleshoot/error-messages/cannot-open-add-in-from-localhost).
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[fe14777](https://github.com/OfficeDev/office-js-docs-pr/commits/fe1477746fc989906b8c20b2d38d351d31faeb1f)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/browsers-used-by-office-web-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/edge-troubleshooting/docs/concepts/browsers-used-by-office-web-add-ins.md) | :warning:Warning | | [Details](#user-content-ffbf8e59-1260be6f)

<a id="ffbf8e59-1260be6f"></a>
### [docs/concepts/browsers-used-by-office-web-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/edge-troubleshooting/docs/concepts/browsers-used-by-office-web-add-ins.md)
 - **Line 60**: **[Warning]** `````Invalid file link:(~/../../troubleshoot/error-messages/cannot-open-add-in-from-localhost).`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C9%5C26%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201909260438129689-1328%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=TSvhFhtqZIMpthFnWt%2FNT6clOHWYhRMwzSsK2dMyiEU%3D&st=2019-09-26T04%3A34%3A24Z&se=2019-10-27T04%3A39%3A24Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[8d3fb17](https://github.com/OfficeDev/office-js-docs-pr/commits/8d3fb176ef61119ce592a197a3c230e649bfc95e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/browsers-used-by-office-web-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/edge-troubleshooting/docs/concepts/browsers-used-by-office-web-add-ins.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C9%5C26%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201909260438387047-1328%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=ShBEXiPX%2FIN3ZH9UcA46yq%2FjJxTYQz4UListCHqMY%2BM%3D&st=2019-09-26T04%3A35%3A45Z&se=2019-10-27T04%3A40%3A45Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**